src/components/utils/weatherVariables.jsx

```
@@ -1,142 +1,175 @@
+import {
+  BsFillSunFill,
+  BsFillCloudRainHeavyFill,
+  BsFillCloudDrizzleFill,
+  BsFillCloudLightningRainFill,
+  BsFillCloudSnowFill,
+  BsCloudFog,
+  BsFillCloudRainFill,
+  BsFillCloudsFill,
+  BsFillCloudSunFill,
+  BsFillCloudHazeFill,
+} from 'react-icons/bs'; // * Sunny, Heavy Intensity Rain, Drizzle, Thunder and Rain, Snow, Fog, Light Rain, Overcast Clouds, Scattered Clouds, Haze
```

ESLint

[error] 'react-icons/bs' imported multiple times.

(import/no-duplicates)